dali/image/CMakeLists.txt

```diff
@@ -22,3 +22,8 @@ if (NOT BUILD_LIBTIFF)
         ${DALI_SRC_DIR}/dali/image/tiff_libtiff.cc
     )
 endif()
+
+set(DALI_SRCS ${DALI_SRCS}
```

Comment:
Note, `dali/image` is meant to be removed at some point. this is only temporary.